app/assets/javascripts/backbone/views/breakeven_product_total_view.coffee

```diff
@@ -0,0 +1,104 @@
+class MS.Views.BreakevenProductTotalView extends Backbone.View
+
+  # The view is called from the editiable table view.
+
+  # events:
+  #   'click tr [data-action="delete"]': 'removeRow'
+  #   'click .actions [data-action="add"]': 'addRow'
+
+  initialize: (options) ->
+    @products = options.products
+
+    @totalFixedCosts = parseFloat(@$('.total_fixed_costs').val())
+
+    _.each @products, (product) =>
+      product.on "product:changed", () =>
+        @calculateTotals()
+
+    # Defer loading of totals until the dom is ready
+    $ =>
+      @calculateTotals()
+
+  calculateTotals: ->
+    @notifyProducts()
+
+    totalPrice = @totalPrice()
+    totalCost = @totalCost()
+    totalQuantity = @totalQuantity()
+    totalRevenue = @totalRevenue()
+    totalPercentageOfSales = @totalPercentageOfSales()
+    totalProductCost = @totalProductCost()
+    Q = @Q()
+
+    totals = {totalPrice, totalCost, totalQuantity, totalRevenue, totalPercentageOfSales, totalProductCost, @totalFixedCosts, Q}
+    @writeTotalsToDom(totals)
+
+  totalPrice: ->
```

Comment:
In future, methods like this that are almost total copies of each other should be abstracted. I bet copy and paste was used to write these methods. That is a big warning sign! The only thing that differs across each is the method called on `product`. I am not happy about this but there isn't time to refactor now unless you think you can do it quickly.